pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/readers/forward/FixedBitMVForwardIndexReader.java

```diff
@@ -101,6 +101,13 @@ public Context createContext() {
     return new Context();
   }
 
+  @Override
```

Comment:
I am unsure if this is the correct approach, but after adding this method, I can get the failing query running locally. Here is the snapshot for reference 
<img width="1114" alt="image" src="https://github.com/apache/pinot/assets/1076944/ee86ebf6-cd1d-4e2b-8053-01e7fb215b06">


I will add the required test cases, based on the rightness of the solution. Thanks